Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2774,33 +2774,6 @@ extension BrowserViewController {
     }
 
     func trackNotificationPermission() {
-        let center = UNUserNotificationCenter.current()
-        center.getNotificationSettings { settings in
-            var authorizationStatus = ""
-            switch settings.authorizationStatus {
-            case .authorized: authorizationStatus = "authorized"
-            case .denied: authorizationStatus = "denied"
-            case .ephemeral: authorizationStatus = "ephemeral"
-            case .provisional: authorizationStatus = "provisional"
-            case .notDetermined: authorizationStatus = "notDetermined"
-            @unknown default: authorizationStatus = "notDetermined"
-            }
-
-            var alertSetting = ""
-            switch settings.alertSetting {
-            case .enabled: alertSetting = "enabled"
-            case .disabled: alertSetting = "disabled"
-            case .notSupported: alertSetting = "notSupported"
-            @unknown default: alertSetting = "notSupported"
-            }
-
-            let extras = [TelemetryWrapper.EventExtraKey.notificationPermissionStatus.rawValue: authorizationStatus,
-                          TelemetryWrapper.EventExtraKey.notificationPermissionAlertSetting.rawValue: alertSetting]
-
-            TelemetryWrapper.recordEvent(category: .action,
-                                         method: .view,
-                                         object: .notificationPermission,
-                                         extras: extras)
-        }
+        NotificationManager().getNotificationSettings { _ in }
```

Comment:
Hmm, it looks like a singleton but doesn't behave like one. 

@thatswinnie do you want this class to be instantiated every time you request something?

Reply:
Yes, we don't use it in a lot of places. The alternative would be to make the methods static but then I'd need to pass in an instance of UserNotificationCenterProtocol to each method and some of the methods already have too many parameters for my taste.

Reply:
Wondering if making this a singleton would be beneficial or can we leverage DIP?

Reply:
Actually, it isn't required in form of a singleton so disregard my last statement